_partials/self-hosted/_login-banner.mdx

```diff
@@ -0,0 +1,66 @@
+---
+partial_category: self-hosted
+partial_name: login-banner
+---
+
+
+
+
+{props.edition} allows you to add a Login banner, also known as an Authority to Operate (ATO) banner or System Message, to the login page. This banner can display important information to users before they log in to the system console or an individual tenant.
+
+
+:::warning
+
+Setting a login banner at the system console will override any tenant-specific login banners configured. Refer to the <VersionedLink text="Tenant Login Banner" edition={props.edition}  url="/tenant-settings/login-banner/"/> guide to learn more about tenant-specific login banners.
```

Comment:
Does it make sense to use edition={props.edition} inside the versioned link? :B

Reply:
Yup, not needed.